adaptors/plugins.js

```diff
@@ -1,4 +1,6 @@
+"use strict";
+
 module.exports = [
-  require('lout'),
-  require('hapi-auth-bearer-token')
+  require(`lout`),
```

Comment:
I think this is how it is passed into Hapi (directly from what is exported)? I'm not sure, I didn't change the logic here.